strawberry/exceptions.py

```diff
@@ -240,3 +240,11 @@ def __init__(self):
             "that inherits from BaseContext or a dictionary"
         )
         super().__init__(message)
+
+
+class PersistedQueryNotFoundError(Exception):
+    message: str = "PersistedQueryNotFound"
+    code: str = "PERSISTED_QUERY_NOT_FOUND"
+
+    def __init__(self):
+        super().__init__(self.code)
```

Comment:
Should the `super()` call provide a message that's more like a sentence? What's the reasoning for using `code` as the exception message (not to be confused with the `.message` here)

Reply:
we also aren't settings code and message anywhere, right?